common/components/Header/index.jsx

```diff
@@ -22,7 +22,7 @@ export default class Header extends Component {
 
     return (
       <div>
-        <section className="bg-gradient header-branding">
+        <section className="bg-gradient header-branding {selectedNode.network}">
```

Comment:
The class name feels too generic, does attaching a `branding-node-{selectedNode.network}` sound good?